staging/src/k8s.io/apiserver/pkg/server/options/recommended.go

```diff
@@ -77,3 +77,27 @@ func (o *RecommendedOptions) ApplyTo(config *server.Config) error {
 
 	return nil
 }
+
+func (o *RecommendedOptions) Validate() []error {
```

Comment:
Godoc is fine. But please don't repeat what is checked in detail there. That will get out of date very soon.